test/cloudtest/test_computed_shared_fate.py

```diff
@@ -88,6 +88,8 @@ def validate(mz: MaterializeApplication, seed: int) -> None:
         ),
         no_reset=True,
         seed=seed,
+        # Longer timeout to account for CrashLoopBackOff
+        default_timeout=1200,
```

Comment:
So it may take up to 1200 seconds = 20 minutes for the cluster to reconstitute itself?